includes/Story_Post_Type.php

```diff
@@ -579,6 +594,7 @@ public function get_editor_settings() {
 		$mime_types       = $this->get_allowed_mime_types();
 		$mime_image_types = $this->get_allowed_image_mime_types();
 
+		$page_templates_rest_base = $this->get_post_type_rest_base( Page_Template_Post_Type::POST_TYPE_SLUG );
```

Comment:
@spacedmonkey Had to add a separate function since the PhpStan linter gave an error if this method had another `if` clause in it (for NPath complexity). Please check this part again and feel free to make changes directly to the PHP file.

Reply:
@miina I broke this out into a new PR. https://github.com/google/web-stories-wp/pull/7304

In short, when reviewing this, saw LOTS of repeated code all over the place that was very messy. Decided to refactor to make the code a little cleaner.

Reply:
Great, thanks!